src/test/java/io/openio/sds/ClientITest.java

```diff
@@ -539,6 +543,25 @@ public void handleSpecificObjectVersion() throws IOException,
                 Assert.assertEquals(
                         Hex.toHex(MessageDigest.getInstance("MD5").digest(src)),
                         oinf.hash());
+
+                client.setObjectProperties(url, 123456789L, props);
+                Map<String, String> res = client.getObjectProperties(url);
```

Comment:
The 2 `getObjectProperties` calls are executed without specifying the `version` argument, is that wanted?